docs/concepts.md

```diff
@@ -18,10 +18,25 @@ Its interface should be familiar to users of MySQL, Postgres, Oracle, Hive, Pres
 
 The KSQL CLI acts as a client to the KSQL server (see next section).
 
+>An alternative to the KSQL CLI is to use the KSQL Experimental UI.
+
+### KSQL Experimental UI
+The KSQL Experimental UI is a web interface for KSQL. It runs on every KSQL Server unless it is disabled. To view the status and avialability of the page visit **http://ksqlserver:8080**
```

Comment:
Suggestion:

> KSQL currently ships with [an experimental web UI](https://github.com/confluentinc/ksql-experimental-ui), whose goal it is to provide a simple visual wrapper for the interactive KSQL CLI. The experimental UI is meant to be used for local development, testing, and demoing KSQL.  If you want to use KSQL interactively in production environments, we recommend the use of the KSQL CLI instead.
>
> * By default, the KSQL Experimental UI runs on every KSQL Server and is accessible at http://ksqlserver:8080/index.html.
> * To disable the experimental UI you must add `ui.enabled=false` to the server's configuration file `ksqlserver.properties`.
> * The UI shares the port with KSQL's REST API, which you can configure via the `listeners` config option.

Reply:
@bluemonk3y : Didn't you suggest the following change:

* The current "version" info endpoint at http://localhost:8080 (that shows the JSON snippet `{"KsqlServerInfo":{"version":"0.4" ...}`) should be moved to http://localhost:8080/status.
* The experimental UI should be accessible on http://localhost:8080 rather than on http://localhost:8080/index.html

?

Reply:
@miguno  - I wanted to apply the minimal code change - keeping the ServiceInfo at the root is more akin to REST standards. The compromise of not auto-redirecting to /index.html is achieved by displaying the UI URL in the Status json so the user is aware. Yes, this isnt an ideal UX - I can change it if you really think its worthwhile ;)

Reply:
> keeping the ServiceInfo at the root is more akin to REST standards.
Is it?  Not sure there's a standard or even a best practice. :-)  But I'm ok to keep the service info at the root. 
 However, if we do that then I suggest to change the UI from `/index.html` to `/ui.html` or sth similar.  Otherwise I think it is really confusing as most people, I believe, would expect that `/` and `/index.html` would be identical.

WDYT?

Reply:
docroot - is a html-rest microservice thing - where it responds with important top-level info about the svc etc anyways - I think I'm stuck - I see your point, let me look at redirecting again....